salt/orch/update.sls

```diff
@@ -593,6 +610,17 @@ unset-{{ grain }}-grain:
     - require:
       - reenable-transactional-update-timer
 {%- endfor %}
+{%- elif enabled_timer_nodes_list|length > 0 %}
+reenable-transactional-update-timer:
+  salt.function:
+    - tgt: '{{ enabled_timer_nodes_list|join(",") }}'
```

Comment:
Why can't we just enable the transactional update timer in all the nodes in the cluster?

Reply:
That's one of the requirements of the bug: we should only reenable the timer where it was already enabled before the update. In other words, if the admin manually disabled the timer, we shouldn't reenable it after the update.